src/main/tray/create-tray-icon.injectable.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { nativeImage } from "electron";
+import type { NativeImage  } from "electron";
+import { base64 } from "../../common/utils";
+import sharp from "sharp";
+import { JSDOM } from "jsdom";
+import LogoLens from "../../renderer/components/icon/logo-lens.svg";
+import Notice from "../../renderer/components/icon/notice.svg";
+import { getInjectable } from "@ogre-tools/injectable";
+import isMacInjectable from "../../common/vars/is-mac.injectable";
+
+export interface CreateTrayIconArgs {
+  useDarkColors: boolean;
+  updateAvailable: boolean;
+}
+
+export type CreateTrayIcon = (args: CreateTrayIconArgs) => Promise<NativeImage>;
+
+const createTrayIconInjectable = getInjectable({
+  id: "create-tray-icon",
+  instantiate: (di): CreateTrayIcon => {
+    const produceTemplateImage = di.inject(isMacInjectable);
+
+    return async ({ useDarkColors, updateAvailable }) => {
+      const size = 32; // 2x zoom
+      const trayIconColor = useDarkColors ? "white" : "black"; // Invert to show contrast
```

Comment:
Necessity of this line should be double checked because tray icon in latests macOS version should **always** be dark no matter what native theme is used.

For example, here is one of the installed apps changed tray color to white for no good reason.
<img width="513" alt="dark tray icons" src="https://user-images.githubusercontent.com/9607060/166941237-02ff668e-a28f-4887-a7b7-1e7f824a1492.png">

Reply:
Yes I know, that is what `produceTemplateImage` is supposed to cover.